Target-MVD-Mathias/Extensions/UIStoryboardExtension.swift

```diff
@@ -17,4 +17,10 @@ extension UIStoryboard {
   func instantiateViewController <T: UIViewController>(_ type: T.Type) -> T? {
     return instantiateViewController(withIdentifier: String(describing: type)) as? T
   }
+  
+  class func instantiateInitialViewController(storyboardIdentifier: String = "Main", bundle: Bundle? = nil) -> UIViewController? {
+    let storyboard = UIStoryboard(name: storyboardIdentifier, bundle: bundle)
+    return storyboard.instantiateInitialViewController()
```

Comment:
why not use `UIStoryboard(name: storyboardIdentifier, bundle: bundle).instantiateInitialViewController() directly?

Reply:
Because I wanted to be consistent with the first function of this extension